acq4/devices/VimbaXCamera.py

```diff
@@ -285,10 +291,12 @@ def _featureNameToParamName(name):
 
 
 def main():
-    class MockManager:
-        def declareInterface(self, *args, **kwargs):
-            pass
-    cam = VimbaXCamera(MockManager(), {'id': 'DEV_000F315B9827'}, 'test')
+    import pyqtgraph as pg
+    from acq4.Manager import Manager
+
+    pg.mkQApp()  # for event loop
+    id_ = VimbaXCamera.listCameras()[0]
+    cam = VimbaXCamera(Manager(), {'id': id_}, 'test')
```

Comment:
okay, so the refactors of Manager above are to make it so I can instantiate one here in a test.